### PR TITLE
SEQNG-250: Support sending commands through GIAPI

### PR DESCRIPTION
**Pull request by cquiroz**
This is the next step on GIAPI support with the ability to send commands.

### CODE REVIEW

File: modules/edu.gemini.giapi.client/src/main/scala/edu/gemini/giapi/client/commands.scala [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        command,
+        (hr: HandlerResponse, _: Command) => {
+          if (hr.getResponse === Response.ERROR || hr.getResponse === Response.NOANSWER) {
+            cb(Right(Error(hr.getResponse, hr.getMessage)))
+          } else {
+            cb(Right(Completed(hr.getResponse)))
+          }
+          ()
+        },
+        2000
```
Comment (jluhrs) [inline modules/edu.gemini.giapi.client/src/main/scala/edu/gemini/giapi/client/commands.scala]: Magic number ?

Comment (cquiroz) [reply modules/edu.gemini.giapi.client/src/main/scala/edu/gemini/giapi/client/commands.scala]: Good point, this should be configurable

File: modules/edu.gemini.giapi.client/src/main/scala/edu/gemini/giapi/client/gpi/GPIClient.scala [hunk 2]
```diff
@@ -3,13 +3,19 @@
```
Comment (jluhrs) [inline modules/edu.gemini.giapi.client/src/main/scala/edu/gemini/giapi/client/gpi/GPIClient.scala]: This code seems to have the same function for GPI that GmosEpics.scala. Shouldn't it be in seqexec/server/gpi ?

Comment (cquiroz) [reply modules/edu.gemini.giapi.client/src/main/scala/edu/gemini/giapi/client/gpi/GPIClient.scala]: Eventually I"ll move it there

**Pull request merged**